docs/content/usage/integrations/aws-security-hub.md

```diff
@@ -10,41 +10,41 @@ The Infection Monkey integration with the [AWS Security Hub](https://docs.aws.am
 
 ![AWS security hub logo](/images/usage/integrations/AWS-Security-Hub-logo.png "AWS security hub logo")
 
-The integration will send _all_ Infection Monkey findings (typically low tens of findings) to the security hub at the end of a Monkey breach simulation.
+The integration will send all Infection Monkey findings (typically 10 to 40) to the AWS Security Hub at the end of a breach simulation.
 
 ## Setup
 
-If the correct permissions have been set on the AWS IAM role of the Monkey Island machine, then the Island will automatically export its findings to the AWS security hub.
+If the correct AWS IAM role permissions have been set on the Monkey Island machine, it will automatically export its findings to the AWS Security Hub.
 
-### Specific permissions required for security hub
+### Specific permissions required for the AWS Security Hub
 
 - `"securityhub:UpdateFindings"`
 - `"securityhub:BatchImportFindings"`
 
-Note that the integration is specifically between your Monkey Island and the security hub. The Infection Monkey is an free project and there is no centralised infrastructure.
+
+Note that this integration is specifically between your Monkey Island and the AWS Security Hub. The Infection Monkey is a free project, and there is no centralized infrastructure.
 
 ### Enabling finding reception
 
-Before starting the scan, make sure that AWS Security Hub is accepting findings by enabling Infection Monkey
- integration. Find **GuardiCore: AWS Infection Monkey** integration on the list and click on **Accept findings**.
+Before starting the scan, make sure that the AWS Security Hub is accepting findings by enabling the Infection Monkey integration. Find **GuardiCore: AWS Infection Monkey** integration on the list and click on **Accept findings**.
 
 ![Enabled integration](/images/usage/integrations/security-hub-enable-accepting-findings.png "Enabled integration")
 
 ## Integration details
 
-The Infection Monkey reports the following types of issues to the AWS security hub: `Software and Configuration Checks/Vulnerabilities/CVE`.
+The Infection Monkey reports the following types of issues to the AWS Security Hub: `Software and Configuration Checks/Vulnerabilities/CVE`.
 
-Specifically, the Island sends findings for all vulnerabilities it finds along with generic findings on the network (such as segmentation issues). Our normalized severity is 100, while most issues we report range between 1 and 10.
+Specifically, the Infection Monkey sends findings for all vulnerabilities it finds along with generic findings on the network (such as segmentation issues). Our normalized severity is 100, while most issues we report range between one and 10.
```

Comment:
I'm not sure what the style guides say, but "between one and 10" looks weird. Since this is technical documentation, can we stick with numbers?